packages/css/tailwind.config.js

```diff
@@ -16,7 +17,18 @@ const colors = Object.assign(
 );
 
 module.exports = {
-    content: ["../**/*.{vue,html}"],
+    content: [
```

Comment:
Minor (obviously): Do you know if there is a way we could avoid that enumeration of projects (that we could maybe do in a future refactor)? Not that it matters now at all but this seems like a great way to build a new app and not have the CSS working because you forgot to register the app here 😄